BackofficeBundle/Resources/views/layout.html.twig

```diff
@@ -65,6 +65,8 @@
 
     <script data-pace-options='{ "restartOnRequestAfter": true }' src="{{ asset('js/pace.min.js') }} "></script>
 
+    {{ tinymce_init() }}
```

Comment:
@nicolasThal : In my first attempt I need this as some scripts in our all.js required tinymce to be loaded. I think it's no more the case, but as it changes nothing to be loaded before or after our all.js I suggest to let it before. As some of our future bundles should be related to tinymce, it's more logical to load tinymce before execute plugins.